src/ORM/Rule/IsUnique.php

```diff
@@ -53,6 +53,11 @@ public function __invoke(EntityInterface $entity, array $options)
             return true;
         }
 
+        $permitMultipleNulls = true;
+        if (isset($options['permitMultipleNulls'])) {
```

Comment:
Naming the option `allowMultipleNulls` would be better for consistency since "allowFoo" is used at other places in the framework.